### PR TITLE
pulled original jenkins repo in SmartBear forked repo

### PR DESCRIPTION
**Pull request by SmartBearAshutosh**
Forked SmartBear repo is 27 commits behind original jenkins repo

https://github.com/SmartBear/zephyr-enterprise-test-management-plugin/compare/main...jenkinsci%3Azephyr-enterprise-test-management-plugin%3Amaster

Screenshot 2024-07-22 at 2 35 07 AM

### CODE REVIEW

**Comment by geetanshu-smartbear** [comment]
take the latest pull from main, I have already updated branch names in main .

File: README.md [hunk 1]
```diff
@@ -243,14 +243,16 @@ Compatibility between this plugin and Zephyr Enterprise is:
...
 | Plugin | Zephyr Enterprise |
 |--------|-------------------|
-| v2.5   | v7.0 or later     |
+| v2.9   | v8.1 or later    |
+| v2.8   | v7.16 to v8.0.1    |
+| v2.7   | v7.0 to v7.15.1    |
 | v2.4   | v7.0 to v7.15     |
 | v2.3   | v7.0 to v7.15     |
-| v2.2   | v6.7.x to v7.15   |
+| v2.2   | v6.7.x to v7.0    |
```
Comment (d-kafley) [inline README.md:251]: any specific reason why 7.15 has been changed to 7.0??

File: README.md [hunk 2]
```diff
@@ -243,14 +243,16 @@ Compatibility between this plugin and Zephyr Enterprise is:
 
 | Plugin | Zephyr Enterprise |
 |--------|-------------------|
-| v2.5   | v7.0 or later     |
+| v2.9   | v8.1 or later    |
+| v2.8   | v7.16 to v8.0.1    |
+| v2.7   | v7.0 to v7.15.1    |
 | v2.4   | v7.0 to v7.15     |
```
Comment (d-kafley) [inline README.md:249]: version 2.5 seems to be missing have we removed intentionally ??

Comment (SmartBearAshutosh) [reply README.md:249]: perhaps, let me check. I just synced the source and destination

Screenshot 2024-07-22 at 12 04 56 PM

**Review by ArunkumarB729 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SmartBearAshutosh) [reply README.md:251]: This is from a previous commit 
https://github.com/SmartBear/zephyr-enterprise-test-management-plugin/commit/4f4689188ceb1ad717b4e54fb2ce7e7191bc02cb

File: README.md (hunk 2 above)
Comment (SmartBearAshutosh) [reply README.md:249]: @d-kafley 

This is from a previous commit
[4f46891](https://github.com/SmartBear/zephyr-enterprise-test-management-plugin/commit/4f4689188ceb1ad717b4e54fb2ce7e7191bc02cb)

**Review by d-kafley — APPROVED** [review]

**Review by geetanshu-smartbear — APPROVED** [review]

**Pull request merged**